ws-nextjs-app/utilities/cspHeaderResponse/index.ts

```diff
@@ -34,17 +37,60 @@ const directiveToString = (directives: Record<string, string | string[]>) => {
   return cspValue;
 };
 
-const cspHeaderResponse = ({ request }: { request: NextRequest }) => {
+const getToggleDefintions = (
+  toggles: Toggles = {},
+): Record<string, ToggleDefinition> => {
+  const { _environment, ...toggleDefinitions } = toggles;
+  return toggleDefinitions;
+};
+
+const isRelaxedCspEnabled = (
+  omittedCountries: string | number | undefined,
+  country: string,
+): boolean => {
+  if (!omittedCountries || omittedCountries.toString().trim() === '') {
+    return true;
+  }
+
+  const allowedCountries = omittedCountries
+    .toString()
+    .split(',')
+    .map(s => s.trim().toLowerCase())
+    .filter(Boolean);
+
+  return !allowedCountries.includes(country.toLowerCase());
+};
+
+const isValidService = (str: string) => {
+  const [service] = str.split('/').filter(Boolean);
+  return service && services.includes(service as Services);
```

Comment:
This is pretty much the same as [`fallbackServiceParam()`](https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/fetchPageData/utils/getRouteProps/fallbackServiceParam/index.js) but returns a Boolean rather than service/'news'. As this is just a few lines, I decided against making this as part of `fallbackServiceParam`/making `fallbackServiceParam` import this from elsewhere. Happy to refactor if people feel it would be a significant improvement.